core/helpers/EEH_Line_Item.helper.php

```diff
@@ -309,13 +309,15 @@ public static function create_ticket_line_item(EE_Line_Item $total_line_item, EE
         $event = sprintf(_x('(For %1$s)', '(For Event Name)', 'event_espresso'), $first_datetime_name);
         // get event subtotal line
         $events_sub_total = self::get_event_line_item_for_ticket($total_line_item, $ticket);
+        $taxes = $ticket->tax_price_modifiers();
+        $is_taxable = empty($taxes) ? $ticket->taxable() : false;
```

Comment:
so essentially, if the taxes have already been applied, which is obvious by the existence of tax price modifiers, then set `is_taxable` to `false`, otherwise, use the setting for the global flag